kvs-jwt-verify/README.md

```diff
@@ -2,39 +2,48 @@
 
 **CloudFront Functions event type: viewer request**
 
-**Note:** We recommend using the [kvs-jwt-verify example](https://github.com/aws-samples/amazon-cloudfront-functions/tree/main/kvs-jwt-verify) instead. It uses Amazon CloudFront KeyValueStore to store the secret key, and is compatible with JavaScript 2.0 runtime. 
+This function validates a JSON Web Token (JWT) in the query string of the incoming request. It is compatible with JavaScript 2.0 runtime. Using your CloudFront KeyValueStore ID is optional.
```

Comment:
This function validates a JSON Web Token (JWT) in the query string of the incoming request. It is compatible with the CloudFront Functions [JavaScript 2.0 runtime](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/functions-javascript-runtime-20.html) and uses [CloudFront KeyValueStore](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/kvs-with-functions.html) to store the secret. Using your CloudFront KeyValueStore ID is optional.